src/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -85,16 +85,8 @@ private static void InitColorTable()
                     // Retrieving getter of the category property of the UserPreferenceChangingEventArgs.
                     Type argsType = Type.GetType("Microsoft.Win32.UserPreferenceChangingEventArgs, System, Version = 4.0.0.0, Culture = neutral, PublicKeyToken = b77a5c561934e089", throwOnError: false);
                     PropertyInfo categoryProperty = argsType?.GetProperty("Category", BindingFlags.Instance | BindingFlags.Public);
```

Comment:
Nit: This can get the method info directly, without going through PropertyInfo indirection.